src/mame/nintendo/vt1682.cpp

```diff
@@ -5185,6 +5261,21 @@ void vt_vt1682_state::vt_vt1682_sound_map(address_map& map)
 	map(0xfffe, 0xffff).r(FUNC(vt_vt1682_state::soundcpu_irq_vector_hack_r)); // probably need custom IRQ support in the core instead...
 }
 
+uint8_t vt_vt1682_state::vt1682_211c_scuirq_clear_r()
+{
+	m_scu_to_main_irq_active = 0;
+	update_main_interrupts();
+	return 0x00;
+}
```

Comment:
Remember to suppress read side effects for debugger accesses.  Also, should this return `space.unmap()`?

Reply:
I don't think this really exists, it isn't the address used, and the later documentation has a correction which relocates it to the write address actually used by the software we have, I've removed it